apps/web/src/components/app/Queue.test.tsx

```
@@ -0,0 +1,260 @@
+}));
+
+vi.mock('./queue/QueueHeader', () => ({
+  QueueHeader: ({
+    onShowHistory,
+    onToggleQueue,
+  }: {
+    onShowHistory: () => void;
+    onToggleQueue: () => void;
+  }) => (
+    <div data-testid="queue-header">
+      <button onClick={onShowHistory}>Show History</button>
```

Biome (2.4.4)

[error] Provide an explicit type prop for the button element.

(lint/a11y/useButtonType)